tests/integration/api_v3/http/posStakers.test.js

```diff
@@ -0,0 +1,152 @@
+const { api } = require('../../../helpers/api');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV3 = `${baseUrl}/api/v3`;
+
+const { badRequestSchema } = require('../../../schemas/httpGenerics.schema');
+const { goodRequestSchema } = require('../../../schemas/api_v3/staker.schema');
+
+const endpoint = `${baseUrlV3}/pos/stakers`;
+
+describe('Stakers API', () => {
+	let refValidator;
+	let refValidatorAddress;
+	beforeAll(async () => {
+		do {
+			// eslint-disable-next-line no-await-in-loop
+			const { data: [stakeTx] = [] } = await api.get(`${baseUrlV3}/transactions?moduleCommand=pos:stake&limit=1`);
+			if (stakeTx) {
+				// Destructure to refer first entry of all the sent votes within the transaction
+				const { params: { stakes: [stake] } } = stakeTx;
+				refValidatorAddress = stake.validatorAddress;
+			}
+		} while (!refValidatorAddress);
+		const response2 = await api.get(`${baseUrlV3}/pos/validators?address=${refValidatorAddress}`);
+		[refValidator] = response2.data;
+	});
+
+	describe(`GET ${endpoint}`, () => {
+		it('Returns list of stakers when requested for known validator address', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.votes.length).toBeLessThanOrEqual(10);
+		});
+
+		it('Returns list of stakers when requested with known validator address and offset=1', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&offset=1`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.votes.length).toBeLessThanOrEqual(10);
+		});
+
+		it('Returns list of stakers when requested with known validator address and limit=5', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&limit=5`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
+			expect(response.data.votes.length).toBeLessThanOrEqual(5);
+		});
+
+		it('Returns list of stakers when requested with known validator address, offset=1 and limit=5', async () => {
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&offset=1&limit=5`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data.votes.length).toBeGreaterThanOrEqual(0);
+			expect(response.data.votes.length).toBeLessThanOrEqual(5);
+		});
+
+		it('Returns list of stakers when requested for known validator publicKey', async () => {
+			if (refValidator.name) {
+				const response = await api.get(`${endpoint}?name=${refValidator.publicKey}`);
+				expect(response).toMap(goodRequestSchema);
+				expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
+				expect(response.data.votes.length).toBeLessThanOrEqual(10);
+			}
+		});
```

Comment:
I think, this test case needs to be update either to:
```suggestion
		it('Returns list of stakers when requested for known validator name', async () => {
			if (refValidator.name) {
				const response = await api.get(`${endpoint}?name=${refValidator.name}`);
				expect(response).toMap(goodRequestSchema);
				expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
				expect(response.data.votes.length).toBeLessThanOrEqual(10);
			}
		});
```
or
```suggestion
		it('Returns list of stakers when requested for known validator publicKey', async () => {
			if (refValidator.publicKey) {
				const response = await api.get(`${endpoint}?publicKey=${refValidator. publicKey}`);
				expect(response).toMap(goodRequestSchema);
				expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
				expect(response.data.votes.length).toBeLessThanOrEqual(10);
			}
		});
```